server.R

```diff
@@ -47,19 +47,30 @@ server <- function(input, output) {
   output$plot <- renderPlot({
     
     req(is.numeric(input$map_click[["lng"]]))
+    req(input$wv_labels)
     
     loc <- clicked_coords()
 
     vals <- 
       extract(raster, loc, cell = T) %>% 
-      rename_all(~stringr::str_extract(., "cell$|[0-9]+$")) %>% 
+      dplyr::select(cell, matches("[.0-9]+$")) %>%
+      rename_all(~stringr::str_extract(., "cell$|[.0-9]+$")) %>% 
       tidyr::pivot_longer(cols = -cell) %>% 
-      rename(band = name, reflectance = value) %>% 
-      mutate(
+      rename(band = name, reflectance = value) 
+      
+    if (input$wv_labels == "Sequential"){
+      vals<-mutate(vals, 
```

Comment:
style guides indicate that it should be `x <- y` with whitespace around the operator

Reply:
ok will correct